content/en/docs/Updating/_index.md

```diff
@@ -6,26 +6,49 @@ description: >
  Keeping the theme up to date.
 ---
 
-We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the theme submodule yourself as follows:
+We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). 
+If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the Docsy theme
+yourself. 
 
-1. In your local copy of your project, run:
+Updating Docsy means that your site will build using the latest version of Docsy at `HEAD` and include 
+all the new commits or changes that have been merged since the time when you first added the Docsy submodule, 
+or last updated. If you have any modified content like a Docsy [customiztion](/docs/adding-content/lookandfeel/), those 
+changes will remain in your project. For details about what has changed, see the list of 
+[Docsy commits](https://github.com/google/docsy/commits/master).
+
+## Update a Docsy clone
```

Comment:
Maybe move this below the "updating a submodule", as we're recommending that people do the submodule version for easy updates, or at least updates that won't potentially involve weep-inducing merge madness.